examples/undocumented/python/kernel_sparse_gaussian.py

```diff
@@ -7,11 +7,10 @@
 parameter_list = [[traindat,testdat,1.1],[traindat,testdat,1.2]]
 
 def kernel_sparse_gaussian (fm_train_real=traindat,fm_test_real=testdat,width=1.1 ):
-	from shogun import SparseRealFeatures
 	import shogun as sg
 
-	feats_train=SparseRealFeatures(fm_train_real)
-	feats_test=SparseRealFeatures(fm_test_real)
+	feats_train=sg.create_features(fm_train_real)
+	feats_test=sg.create_features(fm_test_real)
```

Comment:
@karlnapf in this case we aren't loading the matrix as SparseFeatures, because it isn't a sparse scipy matrix... Should there be `sg.create_sparse_features`?

Reply:
Think so yes. And it should be overloaded to take both a sparse matrix but also a dense matrix (for testing etc)

Reply:
OK, makes sense. The issue I have now is that SGSparseMatrix is not part of the visitor pattern atm.. but working on it in another PR.